src/content/release-notes.mdx

```diff
@@ -5,6 +5,36 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.18.0
+
+27 January 2024
+
+This version is compatible with Kubernetes versions 1.25 to 1.28.
+
+### New Features
+
+- Implementation of [Admin Access Tokens](1.18/admin/dashboard/#admin-access-tokens)  <!-- 7636, 7630, 7651 -->
+
+### Improvements
+
+- Added [instructions for deploying Okteto with ArgoCD](1.18/self-hosted/install/argocd/)
+- Change default value of [ingress.forceIngressSubdomain](1.18/reference/helm-chart-values/#ingress) to false. By default, Okteto won't force deployed applications to match [Okteto's subdomain](https://www.okteto.com/docs/1.18/get-started/deploy-your-app/endpoints/)    <!-- 7718 -->
+- Enable by default cluster policy for registry. Enables any of your Okteto users to have read access to any image built in other namespaces. [To limit this to the previous behavior](https://www.okteto.com/docs/1.18/reference/helm-chart-values/#registry), set `registry.pullPolicy` to `namespace`  <!-- 7829 -->
+- The Okteto CLI used for Pipeline installation has been upgraded to [version 2.25.2](https://github.com/okteto/okteto/releases/tag/2.25.0)
+- Upgrade BuildKit to `0.12.5` <!-- 7830 -->
+
+### Bug Fixes
+
+- Hide clear button if namespace search input is empty  <!-- 7667 -->
+- In deploy dialog, fix wrong filter when canceling current repository on SelectRepository  <!--7668 -->
+- Fix filtered namespace view where parent resources were not expanded  <!-- 7669 -->
+- Fix closing create access token dialog from keyboard  <!-- 7671 -->
+- Fix Scale to Zero Period showing wrong value in Previews UI  <!-- 7725 -->
+- Fix wrong "Waking Up" status when switching to a "Sleeping" namespace  <!-- 7817 -->
+- Properly render `code` nodes as inline elements inside a list item  <!-- 7821 -->
```

Comment:
Maybe we could add more detail indicating that this is for the customized readmes for external resources and possibly a link to https://www.okteto.com/docs/1.18/manifest/external-resources/#customizing-your-external-resource-ui-elements

Reply:
Is that for all 4 of these items @ifbyol

Reply:
No, only for the one in the line 34 (`Properly render `code` nodes as inline elements inside a list item`)